### PR TITLE
10m generate

### PR DESCRIPTION
**Pull request by jesrah**
Main changes:
-circleCI, gnomon and faker packages have been added
-database/generateData.js is the main file of interest. It is creating a JSON populated with faker-generated data.
-no tests but they will be added asap (likely sunday/monday).

### CODE REVIEW

File: database/seeder.js [hunk 1]
```diff
@@ -59,4 +60,30 @@ getFullData = (places) => {
...
+//     }
+//     else {
+//       const inputData = JSON.parse(data);
+//       console.log('This is the data:', inputData)
+//       mongodb.Places.create(inputData);
+//     }
+//   });
+// };
+
+// insertData(data);
```
Comment (StevenPurn) [inline database/seeder.js:89]: Should try to avoid pushing commented out files

File: database/generateData.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+const place = {};
+faker.locale = 'en_US';
+//count down from 10m
+const total = 10000000;
+
+const createSidebars = (i) => {
+	if (i === 1) {
+		console.log('record', i, 'created');
+	};
+  place.id = i;
```
Comment (StevenPurn) [inline database/generateData.js:13]: Might be cleaner to assign them all in a single object block.

**Review by StevenPurn — APPROVED** [review]

File: database/seeder.js (hunk 1 above)
Comment (jesrah) [reply database/seeder.js:89]: Will clean up

File: database/generateData.js (hunk 2 above)
Comment (jesrah) [reply database/generateData.js:13]: Agreed, will consider refactoring.

**Pull request merged**